src/apic-extension/azext_apic_extension/aaz/latest/apic/api/_create.py

```diff
@@ -22,6 +22,7 @@ class Create(AAZCommand):
 
     :example: Create API with custom properties
         az apic api create -g contoso-resources -n contoso --api-id echo-api --title "Echo API" --type REST --custom-properties '{\"public-facing\":true}'
+        az apic api create -g contoso-resources -n contoso --api-id echo-api --title "Echo API" --type REST --custom-properties '@customProperties.json'
```

Comment:
@blackchoey Would you like to put the new example here or to create another new example title?

Reply:
Let's create a new example to show the use of `@filename` syntax, so users can clearly know it's about reading the json content from a file. Maybe the title can be: Create API with custom properties from a file (feel free to adjust the tile as you wish).